src/components/Header/Header.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import {
+  AppBar,
+  Grid,
+  IconButton,
+  SwipeableDrawer,
+  Toolbar,
+} from '@mui/material';
+import { useTheme, styled } from '@mui/material/styles';
+import { useMediaQuery } from '@mui/material';
+import MenuIcon from '@mui/icons-material/Menu';
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import Button from '../Button';
+
+export interface HeaderProps {
+  logo: string;
+  appBarColor: 'default' | 'inherit' | 'primary';
+  diclaimerClick: () => void;
+  logoWidth: string;
+  logoHeight: string;
+  signInClick?: () => void;
+  signUpClick?: () => void;
+  disclaimerEnd?: boolean;
+  disclaimerStart?: boolean;
+  mobileLogoWidth?: string;
+  mobileLogoHeight?: string;
+  withLoginAndSignup?: boolean;
+}
+
+const DrawerHeader = styled('div')(({ theme }: any) => ({
```

Comment:
Why is there an "any" here? If I remove it the type gets automatically recognized as "Theme".

Reply:
That's a good catch! Fixed